index.html

```diff
@@ -5,6 +5,93 @@
     <link rel="stylesheet" href="./assests/styleSheets/indexStyle.css" />
   </head>
   <body>
-    <script type="text/javascript" src="./assests/javaScript/index.js"></script>
+    <main>
+      <header><h1 class="web-title">Movies</h1></header>
+      <ul class="movie-box-container">
+        <a href="./movieDetail.html">
```

Comment:
the first element after `ul` has to be `li`. it cannot be `a`

Reply:
yes after I validate the html in html validator I got and fix that.
I added this after I made other page so I forgot to remove it from this commit.